### PR TITLE
Fixes #334 -- give the assignment PRs a useful commit message

### LINKED ISSUE #334: assign-ids: incorporate advisory ID in title when opening PR
Previously (i.e. when I did it manually) the RUSTSEC ID was used at as the branch name when opening PRs, and also included in the commit message.

Bonus points: it also used to reference the PR which created the advisory as a link in the commit message.

### PR DESCRIPTION
**Pull request by alex**
a) Won't work until there's a release on rustsec-admin
b) Untested, but I feel like there's a decent chance it works!

### CODE REVIEW

File: .github/workflows/assign-ids.yml [hunk 1]
```diff
@@ -15,22 +15,24 @@ jobs:
       uses: actions/cache@v1
       with:
         path: ~/.cargo/bin
-        key: rustsec-admin-v0.2.0
+        key: rustsec-admin-v0.3.0
```
Comment (tarcieri) [inline .github/workflows/assign-ids.yml:18]: FWIW I just opened a PR calling it v0.2.1:

https://github.com/RustSec/rustsec-admin/pull/63

Comment (alex) [reply .github/workflows/assign-ids.yml]: what I get for guessing

Comment (tarcieri) [reply .github/workflows/assign-ids.yml]: It's published now: https://crates.io/crates/rustsec-admin/0.2.1

**Comment by alex** [comment]
@tarcieri were you approving so I would merge my own PR?

**Comment by tarcieri** [comment]
Sure

**Comment by alex** [comment]
Will keep my eyes peeled for the next PR to merge and make sure the result is generated correctly.

**Pull request merged**